carbons/carbons.go

```diff
@@ -63,3 +66,45 @@ func DisableIQ(ctx context.Context, s *xmpp.Session, iq stanza.IQ) error {
 	)), &v)
 	return err
 }
+
+type Received struct {
```

Comment:
Since this struct doesn't actually contain any useful information I think we can just drop it for now. If we decide we need it later we can always add it back.